imgp

```diff
@@ -445,6 +446,8 @@ def parse_args(args=None, namespace=None):
            help='output resolution in HxV or percentage')
     addarg('-o', '--rotate', type=int, metavar='deg',
            help='rotate clockwise by angle (in degrees)')
+    addarg('-q', '--quality', type=int, metavar='N',
+           help='save the image with the specified quality factor N(on a scale of [1, 95]) [default: 75]')
```

Comment:
1. Please shorten option help to

`quality factor (scale 1-95, JPEG-only) [default: 75]`

Update accordingly in README too.

You can also use argparse to limit the input here. Check https://github.com/jarun/Buku/blob/master/buku.py#L2918.